flink-ml-lib/src/test/java/org/apache/flink/ml/classification/linear/LogisticRegressionTest.java

```diff
@@ -0,0 +1,291 @@
+                    Row.of(new DenseVector(new double[] {12, 2, 3, 4}), 1., 2.),
+                    Row.of(new DenseVector(new double[] {13, 2, 3, 4}), 1., 3.),
+                    Row.of(new DenseVector(new double[] {14, 2, 3, 4}), 1., 4.),
+                    Row.of(new DenseVector(new double[] {15, 2, 3, 4}), 1., 5.));
+
+    private static final double[] expectedCoefficient =
+            new double[] {0.528, -0.286, -0.429, -0.572};
+
+    private static final double TOLERANCE = 1e-6;
+
+    private Table bionomialDataTable;
+
+    private Table multinomialDataTable;
+
+    @Before
+    public void before() {
+        Configuration config = new Configuration();
+        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
+        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
+        env.setParallelism(4);
+        env.enableCheckpointing(100);
+        env.setRestartStrategy(RestartStrategies.noRestart());
+        tEnv = StreamTableEnvironment.create(env);
+        Collections.shuffle(bionomialTrainData);
+        bionomialDataTable =
+                tEnv.fromDataStream(
+                        env.fromCollection(
+                                bionomialTrainData,
+                                new RowTypeInfo(
+                                        new TypeInformation[] {
+                                            TypeInformation.of(DenseVector.class),
+                                            Types.DOUBLE,
+                                            Types.DOUBLE
+                                        },
+                                        new String[] {"features", "label", "weight"})));
+        multinomialDataTable =
+                tEnv.fromDataStream(
+                        env.fromCollection(
+                                multinomialTrainData,
+                                new RowTypeInfo(
+                                        new TypeInformation[] {
+                                            TypeInformation.of(DenseVector.class),
+                                            Types.DOUBLE,
+                                            Types.DOUBLE
+                                        },
+                                        new String[] {"features", "label", "weight"})));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    private void verifyPredictionResult(
+            Table output, String featuresCol, String predictionCol, String rawPredictionCol)
+            throws Exception {
+        List<Row> predResult = IteratorUtils.toList(tEnv.toDataStream(output).executeAndCollect());
+        for (Row predictionRow : predResult) {
+            DenseVector feature = (DenseVector) predictionRow.getField(featuresCol);
+            double prediction = (double) predictionRow.getField(predictionCol);
+            double[] rawPrediction = (double[]) predictionRow.getField(rawPredictionCol);
```

Comment:
Should the return type of `rawPredictionCol` be `DenseVector` instead of `double[]` for consistency with `featuresCol`?